easybuild/easyconfigs/f/FSL/FSL-6.0.2-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,78 @@
+# This build still relies on the following libraries (or part of them) shipped with FSL:
+#    GDCHART, libprob, libcprob, cprob, newran, newmat
+# Attempts to use externally built libraries failed. Worth trying again in the future...
+#
+# Some tools like icmp are missing since they are built with the fslpython_install.sh script
+# which locally installs Miniconda and creates a virtual environment. This is too messy and
+# an alternative solution is necessary to install them.
+# https://github.com/easybuilders/easybuild-easyconfigs/issues/7899
+# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation#imcp.2Fimglob.2Fimmv_errors_after_installation
+
+name = 'FSL'
+version = '6.0.2'
+local_pyver = '2.7.15'
+versionsuffix = '-Python-%s' % local_pyver
```

Comment:
@casparvl why this unusual way of specifying the Python version?

Reply:
@verdurin You mean, why not use the template pyver?

Good question, I wasn't sure how to solve this properly, but the situation is this:

Python is not a direct dependency of ```FSL```, but of ```VTK```. ```VTK``` is then a dep for ```FSL```. Because Python is not a direct dependency the template (I think it's ```pyver```, right?) is not defined.

Nevertheless, it seemed to make sense to me to propagate the Python suffix of VTK to FSL, since loading FSL would load a python 2, and thus make it incompatible with any python 3.

It's a bit messy, but I don't know a better way to resolve it. Do you?

Reply:
Oh, and to make things even messier: FSL pulls in a virtualenv based on python3, which also get's added to the path (otherwise, the software doesn't work...). I'm not sure... I could also leave the suffix out, but neither solution feels 'clean'.

Reply:
@verdurin Could you let me know which way you prefer, including the python suffix to let ppl now it's pulling in a VTK with python suffix? Or leave it out?

Reply:
@casparvl Just this is fine imho:

```python
versionsuffix = '-Python-2.7.15'
...
dependencies = [
    ...
    ('VTK', '8.1.1', versionsuffix),
```

There's no point in introducing a separate local variable for the Python version, it's mostly about only having to change it in a single place.

Reply:
You're right. The introduction of an extra local variable is a relic from my GPU-build, where the versionsuffix for FSL also includs some ```CUDA-...``` (which should NOT be passed on to VTK of course). I'll do it your way for this PR.